docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -243,6 +243,14 @@ To enable performance hints for your application, use the following code:
 
 @endsphinxdirective
 
+#### AUtomatic Batching
+
+The `ov::hint::PerformanceMode::THROUGHPUT` mode and the `ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT` mode will trigger Auto-Batching (for example, for the GPU device) by default. You can disable it by setting `ov::hint::allow_auto_batching`(false), or change the default timeout value to a large number, e.g. `ov::auto_batch_timeout(1000)`. See [Automatic Batching](./automatic_batching.md) for more details.
```

Comment:
```suggestion
The `ov::hint::PerformanceMode::THROUGHPUT` mode and the `ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT` mode will trigger Auto-Batching (for example, for the GPU device) by default. You can disable it by setting `ov::hint::allow_auto_batching(false)`, or change the default timeout value to a large number, e.g. `ov::auto_batch_timeout(1000)`. See [Automatic Batching](./automatic_batching.md) for more details.
```
Is `ov::hint::allow_auto_batching(false)` a piece of command? If not, we should explicitly say "`ov::hint::allow_auto_batching` to false".

Reply:
Yes. It is a workable C++ statement.